syz-manager/rpc.go

```diff
@@ -511,13 +506,13 @@ func validateRequest(req *queue.Request) error {
 	return nil
 }
 
-func (serv *RPCServer) createInstance(name string, injectLog chan<- []byte) {
+func (serv *RPCServer) createInstance(name string, injectExec chan<- bool) {
 	runner := &Runner{
-		injectLog:  injectLog,
-		injectStop: make(chan bool),
-		shutdown:   make(chan bool, 1),
+		injectExec: injectExec,
+		finished:   make(chan bool),
 		requests:   make(map[int64]*queue.Request),
 		executing:  make(map[int64]bool),
+		lastExec:   MakeLastExecuting(serv.cfg.Procs, 3),
```

Comment:
I think 3 is too small, that's just 12-15 programs. Maybe 5 or 6?

Reply:
Done. 6 it is.